third_party/lua/luadebug.lua

```diff
@@ -0,0 +1,808 @@
+--[[
+    Copyright (c) 2020 Scott Lembcke and Howling Moon Software
+
```

Comment:
I see that almost all changes made in the first patch are just cosmetics: spaces and line breaks. I suggest to remove them, since this module is adopted and placed in `third_party` tree. As a result, the only changes to be left are Tarantool specific: `debugger.lua` renaming, `jit.version` change (as @ligurio suggested below).

As a result our tree will contain only required changes and our copy will differ from the original module only in the changed spots.